cypress/e2e/data-import/log-details/data-update-after-deleting-log-with-user-filter.cy.js

```diff
@@ -0,0 +1,59 @@
+import { DevTeams, TestTypes, Permissions } from '../../../support/dictionary';
+import TopMenu from '../../../support/fragments/topMenu';
+import DataImport from '../../../support/fragments/data_import/dataImport';
+import JobProfiles from '../../../support/fragments/data_import/job_profiles/jobProfiles';
+import Logs from '../../../support/fragments/data_import/logs/logs';
+import LogsViewAll from '../../../support/fragments/data_import/logs/logsViewAll';
+import DeleteDataImportLogsModal from '../../../support/fragments/data_import/logs/deleteDataImportLogsModal';
+import Users from '../../../support/fragments/users/users';
+
+describe('data-import', () => {
+  describe('Uploading files', () => {
+    let user = {};
+    const filesNames = 'C2378_File1.mrc';
+    const fileNameForUpload = 'autoTestC358540.mrc';
+    const jobProfileToRun = 'Default - Create instance and SRS MARC Bib';
+
+    before('create test data', () => {
+      cy.createTempUser([
+        Permissions.moduleDataImportEnabled.gui,
+        Permissions.dataImportDeleteLogs.gui,
+      ]).then((userProperties) => {
+        user = userProperties;
+        cy.login(user.username, user.password, {
+          path: TopMenu.dataImportPath,
+          waiter: DataImport.waitLoading,
+        });
+      });
+      DataImport.verifyUploadState();
+      DataImport.uploadFile(filesNames, fileNameForUpload);
+      JobProfiles.waitLoadingList();
+      JobProfiles.search(jobProfileToRun);
+      JobProfiles.runImportFile();
+      JobProfiles.waitFileIsImported(fileNameForUpload);
+    });
+
+    afterEach(() => {
+      cy.getAdminToken().then(() => {
+        Users.deleteViaApi(user.userId);
+      });
+    });
+
+    it(
+      'C358540 Check the data update in the User filter after deleting the logs on the View all page (folijet) (TaaS)',
+      { tags: [TestTypes.extendedPath, DevTeams.folijet] },
+      () => {
+        Logs.openViewAllLogs();
+        LogsViewAll.viewAllIsOpened();
+        LogsViewAll.openUserIdAccordion();
+        LogsViewAll.filterJobsByUser(`${user.firstName} ${user.lastName}`);
+        cy.wait(2000);
```

Comment:
we need to use either intercepts for waiting for the specific requests or provide comments why we still need to use cy.wait